lib/UserNotificationConfigHandler.php

```diff
@@ -98,7 +98,12 @@ public function getExpirationTimeForNormalNotification() {
 	 * to include the userid owning the password and an id for the password
 	 */
 	public function markAboutToExpireNotificationSentFor(OldPassword $passInfo) {
-		$this->config->setUserValue($passInfo->getUid(), 'password_policy', 'aboutToExpireSent', $passInfo->getId());
+		$this->config->setUserValue(
```

Comment:
Please share the error which is fixed through this

Reply:
``$passInfo->getId()`` returns an ``int`` but ``setUserValue()`` expects a string.